recipes/r-norm/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = '1.0-9.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-norm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: norm_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/norm_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/norm/norm_{{ version }}.tar.gz
+
+  sha256: 305cbf007f3905cfd535ed9bf5ae3e2995e228cc8883d6482e5d3a2f02814106
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('norm')"  # [not win]
+    - "\"%R%\" -e \"library('norm')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/norm/index.html
+  license: file
```

Comment:
This looks wired, and the license information is also not very detailed.

Reply:
So it seems like this package doesn't have a valid license. I'll remove it.